.gitignore

```diff
@@ -1,25 +1,20 @@
 docs/_build
 .ignored
 .idea
-
 # Ignore irrelevant files from the Sublime Text editor
 *.sublime-workspace
 *.sublime-project
-
 # Ignore irrelevant files from the VS code editor
 .vscode/
-
 # Ignore .hgignore for contributors using Mercurial.
 .hgignore
-
 # Ignore files generated during build
 build/
-
 venv/
-
 *.egg-info
-
 __pycache__
-
```

Comment:
Unless there are technical reasons, a content addition should not include (style) changes in the technical setup of the project, IMHO.